.github/workflows/release.yml

```
@@ -0,0 +1,27 @@
+name: release-version
+
+on:
+  push:
+    tags:
+      - v*.*.*
+
+jobs:
+  release-version:
+    name: release-version
+    strategy:
+      matrix:
+        os: [ ubuntu-20.04, macOs-latest, windows-latest ]
```

```suggestion
        os: [ ubuntu-latest, macOs-latest, windows-latest ]
```
There is not need to use 20.04. In addition, 20.04 will become latest soon.